logbook-api/src/main/java/org/zalando/logbook/BaseHttpMessage.java

```diff
@@ -43,7 +43,7 @@
 
     class HeadersBuilder {
 
-        private final Map<String, List<String>> headers;
+        private Map<String, List<String>> headers;
```

Comment:
This one is helper for private usage. I do not see any reason to make double copy of Map only to make it reusable, knowing that we always use it only once. On the other side it is defended against reuse so if you ever will need to reuse it you can just refactor it.